### ISSUE TITLE
Mod Conflict - Crash

### ISSUE THREAD

**Issue by Eliteneet** [gnembon/fabric-carpet#118]
[https://pastebin.com/c0WDhdgL](https://pastebin.com/c0WDhdgL)

I'm having a mixin conflict with carpet and bunch o trinkets. I'm reporting it with both you and the developer of bunch o trinkets.

**Comment by gnembon** [comment]
how did you tell its with that specific mod - didn't see them targeting the same files as carpet does.

**Comment by Eliteneet** [comment]
Process of elimination basically. I add Bunch of Trinkets to my instance and I get that crash, take it out the game launches fine.

Also that mod has an item that increases the players reach. Placing/Breaking blocks.

**Comment by emilyploszaj** [comment]
Hi, Bunch O' Trinkets dev here, I believe this is a conflict with carpet and this library that Bunch O' Trinkets uses (meaning that any other mods that use this library, such as hardhat steve, will also have the conflict): https://github.com/JamiesWhiteShirt/reach-entity-attributes

**Comment by gnembon** [comment]
players reach range its a precious commodity - maybe it should be handled by the fabric API so others can share. Since carpet doesn't rely on it entirely, I put a require = 0 on it and worked out:
```
01:24:18.818
@ModifyConstant conflict. Skipping carpet.mixins.json:ServerPlayerInteractionManager_antiCheatMixin->@ModifyConstant::addDistance(D)D with priority 1000, already redirected by mixins.reach-entity-attributes.json:ServerPlayerInteractionManagerMixin->@ModifyConstant::modifyReachDistance(D)D with priority 1000
01:24:18.826
Starting integrated minecraft server version 1.15.1
```

Anyways - should be resolved in 1.3.3
